Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/Ltc.azsli

```diff
@@ -366,12 +372,42 @@ void LtcQuadEvaluate(
     }
 
     // IntegrateQuadDiffuse is a cheap approximation compared to specular.
-    diffuse = IntegrateQuadDiffuse(v, vertexCount, doubleSided);
+    float diffuse = IntegrateQuadDiffuse(polygon, vertexCount, doubleSided);
+    float specular = 0.0;
 
-    ApplyLtcMatrixToQuad(ltcMat, v, vertexCount);
+    ApplyLtcMatrixToQuad(ltcMat, polygon, vertexCount);
 
     // IntegrateQuadSpecular uses more accurate integration to handle smooth surfaces correctly.
-    specular = IntegrateQuadSpecular(v, vertexCount, doubleSided);
+    specular = IntegrateQuadSpecular(polygon, vertexCount, doubleSided);
+    
+    // Apply BRDF scale terms (BRDF magnitude and Schlick Fresnel)
+    float2 schlick = ltcAmpMatrix.Sample(PassSrg::LinearSampler, ltcCoords).xy;
+    float3 specularRgb = specular * (schlick.x * surface.specularF0 + (1.0 - surface.specularF0) * schlick.y);
+    
+    if(o_clearCoat_feature_enabled)
+    {
+        float2 ltcCoordsCc = LtcCoords(dot(surface.clearCoat.normal, lightingData.dirToCamera), surface.clearCoat.roughness);
+        float3x3 ltcMatCc = LtcMatrix(ltcMatrix, ltcCoordsCc);
+        
+        float3 polygonCc[5];
+
+        // Transform the points of the light into the space of the clearcoat normal's hemisphere.
+        TransformQuadToOrthonormalBasis(surface.clearCoat.normal, lightingData.dirToCamera, p, polygonCc);
+
+        ApplyLtcMatrixToQuad(ltcMatCc, polygonCc, vertexCount);
+        float clearCoatSpecular = IntegrateQuadSpecular(polygonCc, vertexCount, doubleSided);
+
+        // Apply BRDF scale terms (BRDF magnitude and Schlick Fresnel)
+        const float clearCoatSpecularF0 = 0.04;
+        float2 schlickCc = ltcAmpMatrix.Sample(PassSrg::LinearSampler, ltcCoords).xy;
+        float F = schlickCc.x * clearCoatSpecularF0 + (1.0 - clearCoatSpecularF0) * schlickCc.y;
+        F *= surface.clearCoat.factor;
+        diffuse = diffuse * (1.0 - F);
+        specularRgb = (specularRgb * (1.0 - F)) + (clearCoatSpecular * F);
+    }
```

Comment:
Could be wrapped in a function since a lot of this code is close to the existing specular code above.

Reply:
not entirely straightforward because there are a few small differences but I'll see what I can do to dedupe some parts.